web/src/engine/websites/NamiComi.ts

```diff
@@ -0,0 +1,111 @@
+import { Tags } from '../Tags';
+import icon from './NamiComi.webp';
+import { FetchJSON } from '../platform/FetchProvider';
+import { Page } from '../providers/MangaPlugin';
+import { DecoratableMangaScraper, type MangaPlugin, Manga, Chapter } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+type APIResult<T> = {
+    result: string,
+    data: T
+};
+
+type APIManga = {
+    id: string,
+    attributes: {
+        originalLanguage: string,
+        title: {
+            [lang: string]: string
+        }
+    }
+}
+
+type APIChapter = {
+    id: string,
+    attributes: {
+        volume: string,
+        chapter: string,
+        name: null | string,
+        translatedLanguage: string
+    }
+}
+
+type APIPages = {
+    baseUrl: string,
+    hash: string,
+    high: {
+        filename: string
+    }[]
+}
+
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    private readonly apiUrl = 'https://api.namicomi.com';
+    private readonly mangaRegexp = new RegExp(`^${this.URI.origin}/en/title/(\[^/]+)\/[^/]+$`);
```

Comment:
Website is tagged as multilingual, but regex only matches oath `/en/` is this by intention?

Reply:
Yes. Lang is defined in chapters. You can get all languages from the same manga page by clicking on a button (the api returns all chapters tho). I think the en is for the website interface language.

Reply:
This has nothing to do with manga language, its the interface. But yeah, just saw website settings, there are others.